.github/workflows/ci.yml

```
@@ -17,7 +17,9 @@ jobs:
       fail-fast: false
       matrix:
         version:
+          - 'nightly'
           - '1'
+          - '1.3'
```

```suggestion
          - '1.6'
          - '1.3'
```
maybe also test 1.6 since it is the lts?

---

I will add it for now - maybe it is not necessary as testing 1.3 and 1.7 should catch everything in between, but then this package is used quite a lot so maybe better to over test. As far as I know, we can have up to 20 concurrent Github actions. If that becomes an issue we could still restrict the number operating system that we test for some versions.